src/utils/renderContentControls/renderContentControls.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {Fragment} from 'react';
+
+import {Buttons, Links} from '../../components';
+import {ButtonProps, ContentSize, LinkProps} from '../../models';
+import {block} from '../../utils';
+
+import './ContentControls.scss';
+
+const b = block('content-controls');
+
+type ContentControlsArgs = {
+    links?: LinkProps[];
+    buttons?: ButtonProps[];
+    titleId?: string;
+    size?: ContentSize;
+    qa?: Record<string, string>;
+};
+const renderContentControls = (
```

Comment:
```
type ContentControlsProps = {
    links?: LinkProps[];
    buttons?: ButtonProps[];
    titleId?: string;
    size?: ContentSize;
    qa?: Record<string, string>;
   children: (children: React.ReactElement) => React.ReactElement;
};

export const ContentControls = (props: ContentControls) => {
 const {links, buttons, titleId, size = 's', qa = {}, children} = props;
  const {links: linksQa, link: linkQa, buttons: buttonsQa, button: buttonQa} = qa;

  return links || buttons
        ? children(
              <Fragment>
                  <Links
                      className={b('links', {size})}
                      size={size}
                      links={links}
                      titleId={titleId}
                      qa={linksQa}
                      linkQa={linkQa}
                  />
                  <Buttons
                      className={b('buttons', {size})}
                      size={size}
                      buttons={buttons}
                      titleId={titleId}
                      qa={buttonsQa}
                      buttonQa={buttonQa}
                  />
              </Fragment>,
          )
        : null;

}
```

Reply:
The suggested code would be treated as a React Component.  Thus it won't work within CardBase. Please, have a look at https://github.com/gravity-ui/page-constructor/blob/71ca3d9dac4aeee5d3c89710a5fc617693422408/src/components/CardBase/CardBase.tsx#L79